OneCommunityGlobal/HGNRest
# Ramakrishna Teamcode Regex change and select All with No Team code option fix

## Related PRS (if any):
This Backend PR is related to [#2401](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2401) Frontend PR

…

## Main changes explained:
-updated the teamcode regex in necessary file to meet up new requirement to allow the any special characters anywhere in teamcode

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. Please check frontend PR for further instructions
